applications/GeoMechanicsApplication/tests/test_1d_wave_prop_drained_soil_constant_mass_damping.gid/README.md

```diff
@@ -0,0 +1,11 @@
+# 1D wave propagation in a drained soil column, with constant mass and damping
+
+This tests verifies that the ResidualBasedBlockBuilderAnderSolverWithMassAndDamping is used and gives results.
+
+## Setup
+
+This test is a copy of the test 1D wave propagation in a drained soil column. The difference is that mass and damping matrices only are setup once. For that block_builder and prebuild_dynamics are both set to true in ProjectParameters.json.
```

Comment:
It seems it's possible to refer to the other README as a link by including 
`[Other Readme](../test_1d_wave_prop_drained_soil.gid/README.md)`. I think that would be an improvement.

Reply:
done